src/app-simplefin/app-simplefin.js

```diff
@@ -53,7 +60,11 @@ app.post(
     const endDate = new Date(now.getFullYear(), now.getMonth() + 1, 1);
 
     try {
-      const accounts = await getAccounts(accessKey, startDate, endDate);
+      const accounts = await simplefinService.getAccounts(
+        accessKey,
+        startDate,
+        endDate,
+      );
 
       res.send({
         status: 'ok',
```

Comment:
I've gone over the code in the frontend that uses the accounts from the server, and made sure that the variables used there had the same name as the ones in `Account` (from `models/account.ts`)